src-ghc/Pact/Types/RPC.hs

```diff
@@ -42,12 +42,12 @@ instance NFData c => NFData (PactRPC c)
 instance FromJSON c => FromJSON (PactRPC c) where
     parseJSON =
         withObject "RPC" $ \o ->
-            (Exec <$> o .: "exec") <|> (Continuation <$> o .: "yield")
+            (Exec <$> o .: "exec") <|> (Continuation <$> o .: "cont")
     {-# INLINE parseJSON #-}
```

Comment:
Confused why didn't this change already break everything?

Reply:
What do you mean?

Reply:
Aren't the To/FromJSON instances used in the API, yamls, etc?

Reply:
Sorry, meant to say "Why did anything ever work without this change in the main PR for resume"

Reply:
"yield" (or "cont" now) are created by the ToJSON instance (see [here](https://github.com/kadena-io/pact/pull/186/files/f6540132ca5b5ac90cc876ae34fa153ab3eee2da#diff-a789544bfd42f96efbf5288880506f25R50)). And the only time I see toJSON being used with relation to the RPC data type is the `jsonResult` function in [PactService](https://github.com/kadena-io/pact/blob/master/src-ghc/Pact/Server/PactService.hs#L71). At least in the code from the resume PR.